src/core.js

```diff
@@ -213,6 +213,9 @@ $.validator.format = function( source, params ) {
 			return $.validator.format.apply( this, args );
 		};
 	}
+	if ( params === undefined ) {
```

Comment:
I did it on purpose because of this line [1013](https://github.com/jzaefferer/jquery-validation/pull/1644/files#diff-7eb52b366866677666470e019283c8eaR1013) in this PR.

Should I omit it and change the line 1013 with the following:
```js
message: this.defaultMessage( element, { method: "remote", parameters: "" } )
```